gammapy/cube/fit.py

```diff
@@ -24,24 +24,31 @@ class SkyModelMapFit(object):
         Counts cube
     exposure : `~gammapy.maps.WcsNDMap`
         Exposure cube
-    psf : `~gammapy.cube.PSFKernel`
```

Comment:
The PSFKernel should remain in the docstring, no?

Reply:
Yes, I think I removed that by accident.